app/services/gcp_bucket.py

```diff
@@ -53,7 +53,13 @@ def get_gcp_bucket():
 def get_blob(bucket, file_name):
     """ Read data as json (and convert to dict)."""
     if bucket.blob(file_name).exists():
-        json_dict = json.loads(bucket.blob(file_name, chunk_size=262144).download_as_string())
+        # json_dict = json.loads(bucket.blob(file_name, chunk_size=262144).download_as_string())
+        json_dict = [
+            json.loads(item) for item in bucket.blob(file_name, chunk_size=262144)
```

Comment:
do you know of a better way to read a json from GCS? I got the impression that `json.loads(bucket.blob(file_name, chunk_size=262144).download_as_string())` is not a good approach so might be a good opportunity to change this now. perhaps @jontutcher has some advice.

Reply:
also there is one integration test failing because it tries to read from the default data dump file `anonymised_uas_extract_list.json` which doesn't follow the JSON per line format. either the file to be updated or this line to point to a different file https://github.com/bbc/connected-data-pseudocone/blob/daeaa660eb71923a5ce08342fb14d95c8b91b7d4/app/settings.py#L17

Reply:
I think on this step here that we should not read from gcp all the time. I'd rather check if there is a local version of this file, and if not, download it.

Reply:
But maybe removing this bit should be on another ticket.